src/gui/main.cpp

```diff
@@ -68,24 +293,29 @@ int main(int argc, char **argv)
     }
 #endif
 
-    // if the application is already running, notify it.
+#if 0
+    // TODO: upcoming pr introducing KDSingleApplication
     if (app.isRunning()) {
-        qCInfo(lcApplication) << "Already running, exiting...";
+    // if the application is already running, notify it.
+        qCInfo(lcMain) << "Already running, exiting...";
         if (app.isSessionRestored()) {
             // This call is mirrored with the one in Application::slotParseMessage
-            qCInfo(lcApplication) << "Session was restored, don't notify app!";
+            qCInfo(lcMain) << "Session was restored, don't notify app!";
             return -1;
         }
 
         QStringList args = app.arguments();
         if (args.size() > 1) {
             QString msg = args.join(QLatin1String("|"));
-            if (!app.sendMessage(ApplicationPrivate::msgParseOptionsC() + msg))
+            if (!app.sendMessage(msgParseOptionsC() + msg))
                 return -1;
         }
         return 0;
     }
 
+#endif
+
+    // TODO: still needed? Move to platform
```

Comment:
Have you found an answer yet?

Reply:
Nope but out of scope

Reply:
https://github.com/owncloud/client/issues/10761